src/components/Navigation/styles.scss

```diff
@@ -0,0 +1,88 @@
+// Mobile portrait
+.wrapper {
+	width: 100%;
+	height: 100%;
+	padding-top: 10vh;
```

Comment:
```suggestion
```
This seems like a magic number that's going to mess with our vertical centering.

Reply:
Removed and changed height of container to 100%. Seems to be aligning fine now.